gson/src/main/java/com/google/gson/Gson.java

```diff
@@ -410,6 +410,10 @@ private void checkValidFloatingPoint(double value) {
    *  System.out.println("Num JSON reads" + stats.numReads);
    *  System.out.println("Num JSON writes" + stats.numWrites);
    *  }</pre>
+   *  Note that this call will skip all factories registered before @param skipPast. In case of
```

Comment:
If it does, it's new!
http://stackoverflow.com/questions/1667212/reference-a-method-parameter-in-javadoc